### DISCUSSION TITLE
Re latest Ghostscript version and PDF conversion, is it possible to specify -dNEWPDF?

### DISCUSSION THREAD

**Discussion by DynamicLTechnologySolutions** [dlemstra/Magick.NET#1046] [Help] [answered]
As per subject line.

https://ghostscript.com/pdfi.html

Thanks in advance!

**Comment by dlemstra** [comment, marked as answer]
That feature seems pretty new and it looks like this will become the default in the future. So I don't think we will add something for this in the ImageMagick library.

You can however decide to use it yourself by using a patched version of `ConfigurationFiles.Default.Delegates` where you change the command line options for Ghostscript. And then you will need to initialize ImageMagick with that configuration (https://github.com/dlemstra/Magick.NET/blob/main/docs/Initialization.md).

**Comment by DynamicLTechnologySolutions** [comment]
Thanks for the quick response! Will give that a try.
